staking/app/deploy/3_create_governance.ts

```diff
@@ -1,57 +1,66 @@
 import {
-  PROGRAM_VERSION_V2,
-  withCreateGovernance,
   GovernanceConfig,
   VoteTipping,
   VoteThreshold,
   VoteThresholdType,
+  withCreateProgramGovernance,
+  PROGRAM_VERSION,
 } from "@solana/spl-governance";
 import { Transaction, Connection, PublicKey } from "@solana/web3.js";
 import { BN } from "bn.js";
 import { Constants } from "pyth-staking-wasm";
-import { AUTHORITY_KEYPAIR, RPC_NODE } from "./mainnet_beta";
+import { AUTHORITY_KEYPAIR, RPC_NODE } from "./devnet";
 
-import { GOVERNANCE_ADDRESS, REALM_ID, EPOCH_DURATION } from "../constants";
+import {
+  GOVERNANCE_ADDRESS,
+  REALM_ID,
+  EPOCH_DURATION,
+  STAKING_ADDRESS,
+} from "../constants";
 // Actual transaction hash :
 // mainnet-beta : vjUE28suh1yt42aRtsj8mwYpz4zM17WQo4ujfXCDGQ5WK1z5G2JATYvEduh1vdMt2pT9auVLJnoCQMtiyEP3aYC
-// devnet : 3gKKKPGAfV15yV1Ce6Tn9vmwbeRnMHcyrvDxDpPhHAEr6L8VAe4N3rkNizhLGa7cM19xQaJykt6rxjx651fFRqXM
+// devnet (12/11/23): 2N1w4WGrLGsbcTre7yfpNw6FbD2X3uDpmCd3DVPyNv95jLwjt1vDFqdpcEGM9PMXj7RQgW7fJovWd7RHaouFYbmL
 
 async function main() {
   const tx = new Transaction();
 
   let governanceConfig = new GovernanceConfig({
     communityVoteThreshold: new VoteThreshold({
       type: VoteThresholdType.YesVotePercentage,
-      value: 50,
-    }), // 50% of the locked supply
+      value: 10, // 10%
```

Comment:
this seems wrong?

Reply:
maybe change this back if you changed it for devnet to prevent accidents

Reply:
It's ok to start it with 10% and adjust it later